src/mmSimpleDialogs.cpp

```diff
@@ -1017,7 +1017,7 @@ void mmErrorDialogs::InvalidFile(wxWindow *object, bool open)
 
 void mmErrorDialogs::InvalidAccount(wxWindow *object, bool transfer, TOOL_TIP tm)
 {
-    const wxString& errorHeader = _("Invalid Account");
+    const auto errorHeader = wxString("Invalid Account");
```

Comment:
Should be  wxGetTranslation("Invalid Account") ?
or just "Invalid Account"

Reply:
Good point, I don't see an issue with... `const wxString errorHeader = _("Invalid Account"); `